src/functions.php

```diff
@@ -7,6 +7,8 @@
 use Andig\FritzBox\Converter;
 use Andig\FritzBox\Api;
 use \SimpleXMLElement;
+use Symfony\Component\Console\Output\ConsoleOutput;
```

Comment:
please don't do that- it breaks reusability of the function and introduces tight coupling with symfony. If you eant progress, add a callback function instead.

Reply:
Hmmm... sorry - unsure how to do that... can you point me to an example?

Reply:
Check `setProgress` in backend.php:

            if (is_callable($this->callback)) {
                ($this->callback)();
            }